Content.Server/GameObjects/Components/Chemistry/SolutionComponent.cs

```diff
@@ -281,7 +281,26 @@ void IExamine.Examine(FormattedMessage message)
             {
                 if (_prototypeManager.TryIndex(reagent.ReagentId, out ReagentPrototype proto))
                 {
-                    message.AddText($"{proto.Name}: {reagent.Quantity}u\n");
+                    if (inDetailsRange)
+                    {
+                        message.AddText($"{proto.Name}: {reagent.Quantity}u\n");
+                    }
+                    else
+                    {
+                        //This is trash but it shows the general idea 
+                        var colour = proto.SubstanceColor;
+                        var colourIsh = "Red";
+                        if (colour.G > colour.R)
+                        {
+                            colourIsh = "Green";
+                        }
+                        if (colour.B > colour.G || colour.B > colour.R)
```

Comment:
I think this should be an `&&`, since otherwise the RGB colour `255, 0, 1` is blue.

Reply:
Ah, true enough.